fe-code-review/code.js

```diff
@@ -0,0 +1,142 @@
+app.post('/api/extract', upload.single('file'), async (req, res) => {
+    logInfo('POST /api/extract',req.body);
+    logInfo('FILE=',req.file);
+
+    if (req.body) {
+        const file = req.file;
+        const requestID = req.body.requestID;
+        const project = req.body.project;
+        const idUser = req.body.userID;
+        const user = await User.findOne(idUser);
+
+        if (requestID && project && idUser && user) {
+            logDebug('User with role '+user.role, user);
+            if (user.role === 'ADVISOR' || user.role.indexOf('ADVISOR') > -1)
+                return res.json({requestID, step: 999, status: 'DONE', message: 'Nothing to do for ADVISOR role'});
```

Comment:
First, for more readable code it's better to use curly brackets after if statement
Second, why just send json without status? 
Third, souldn't the message be more generic like "No permissions" for example?